.eslintrc-browser.json

```diff
@@ -0,0 +1,34 @@
+{
+  "root": true,
```

Comment:
Can you use tabs in this file? Only `package.json` should be indented by 2 spaces. You might want to copy jQuery's `.editorconfig` to Sizzle.

Reply:
This item is not resolved yet.